apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -32,14 +32,10 @@ class ProgressBubbleSet extends React.Component {
   };
 
   bubbleDisabled = level => {
```

Comment:
sorry to keep at this but a couple small things:
- you could remove this function and just use `this.props.disabled` on line 77 (and any other places it's used). up to you
- if you don't want to remove this function, remove the `level` parameter and the comment on lines 36-37

Reply:
That makes sense.  I made this change.